qiskit/aqua/aqua_globals.py

```diff
@@ -39,24 +39,36 @@ def __init__(self) -> None:
     @property
     def random_seed(self) -> Optional[int]:
         """Return random seed."""
+        warn_variable('aqua.aqua_globals',
```

Comment:
Why can't we use `warn_class` inside of the `init` of `QiskitAquaGlobals`?

Reply:
Because qiskit-terra does "from qiskit import aqua" in its init. This causes all inits under qiskit.aqua to be run in cascade and lots of deprecation msgs to show. QiskitAquaGlobals is created as global object during Aqua init, so if I have the msg there, it will show to whoever does "import qiskit"